### DISCUSSION TITLE
Support for DuckDB

### DISCUSSION THREAD

**Discussion by judiaz** [pawelsalawa/sqlitestudio#4997] [Ideas]
DuckDB is becoming very popular for data analysis and will be nice to use this program with DuckDB.

**Comment by drewbitt** [comment]
https://github.com/pawelsalawa/sqlitestudio/issues/4985#issue-2290694412

**Comment by judiaz** [comment]
https://github.com/pawelsalawa/sqlitestudio/issues/4985#issue-2290694412

Suggestion already made
